metron-interface/metron-alerts/src/app/pcap/pcap-packet-line/pcap-packet-line.component.spec.ts

```diff
@@ -143,38 +132,4 @@ describe('PcapPacketLineComponent', () => {
       "protos": null 
     });
   });
-
-  it('should extract UDP ipSrcPort fields', () => {
```

Comment:
Why were these tests removed?

Reply:
Merge issue. I checked these and the related code change too. Got no idea how could I missed these. Fixing.